packages/react-components/source/scss/library/components/_alerts.scss

```diff
@@ -69,7 +69,7 @@ $colors: (
 .rc-alert-primary-icon {
   flex-grow: 0;
   flex-shrink: 0;
-  margin: $puppet-common-spacing-base * 2;
+  margin: ($puppet-common-spacing-base * 5) / 2;
   margin-right: 0;
 }
```

Comment:
I think what we want is for this margin to be 8px and then for the padding in `.rc-alert-message` to be `6px 8px` instead of just `8px`